fintech-examples/fintech-ui/src/app/bank/settings/settings.component.html

```diff
@@ -15,6 +15,15 @@ <h4 class="text-secondary">List of Accounts</h4>
             <input type="radio" value="{{ loaFromTppWithNewConsent }}" formControlName="loa" id="loa2" />
             <label for="loa2">Discard propably existing consent ONE TIME ONLY.</label>
           </div>
+          <div class="form-check">
+            <input
+              type="checkbox"
+              value="{{ settingsForm.get('cacheLoa') }}"
+              formControlName="cacheLoa"
+              id="cacheLoa"
+            />
+            <label for="cacheLoa">Use protocol cache</label>
```

Comment:
as we are in fintech ui, it would be more clear to write
Use protocol cache of TPP Server

Reply:
ok